WalletWasabi.Backend/Startup.cs

```diff
@@ -107,7 +110,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, Global g
 			app.UseMiddleware<HeadMethodMiddleware>();
 
 			app.UseResponseCompression();
-
+			rates.SetZone($"zone={ZoneLimits.NotificationTokens} rate=10r/m burst=3 nodelay");
```

Comment:
this is OK if my understanding is correct that it only rate limits that NotificationTokens endpoints. CoinJoinClient has periods of requests more frequent than 10r/m per client